jwt-security-server/src/main/java/velog/soyeon/jwt/controller/UsersController.java

```diff
@@ -49,4 +51,15 @@ public String login(UserRequest userRequest) {
         return jwtConfig.createToken(users.getEmail(), Arrays.asList(users.getUserRole().getValue()));
     }
 
+    @GetMapping
+    public List<String> test() {
+        List<String> userEmailList = new ArrayList<>();
+
+        List<UserDTO> userDTOList = usersService.findAll();
+        for (UserDTO userDTO : userDTOList) {
+            userEmailList.add(userDTO.getEmail());
+        }
+        return userEmailList;
```

Comment:
여기 stream 을 사용하면 어떨까요 ??
```suggestion
 userEmailList.findAll().stream().map(UserDTO::getEmail).collect(Collectors.toList());
```
위처럼 더 가독성이 좋게 사용할 수 있을 것 같습니다.